.codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
```

Comment:
Good question, probably not now that we're going to githubflow instead of gitflow